### ISSUE TITLE
Exception thrown when sort by "CP Evolved"

### ISSUE THREAD

**Issue by hkguy80** [Wolfsblvt/BlossomsPokemonGoManager#467]
When I tried  to sort by "CP Evolved", the following error message is prompt and quit the program.

"ClassCasstException: java.lang.Integer cannot be cast to java.lang.String

Application shuts down now.
"

**Comment by jayrifkin** [comment]
I experienced the same error message and crash repeatedly (when trying to sort on few different columns) and was also able to get it to stop happening.

Here's how to reproduce:
1. In the game, rename a pokemon with a space and then a number. ( Example: "Magikarp 91" )
2. Start BPGM, and try to sort by IV.
3. The error message 'java.lang.Integer cannot be cast to java.lang.String' appears and then the application crashes.
4. Going back into the game and removing the space in the name ( Example: "Mk91" ) fixes the problem. The app now handles the sorting without erroring out.

NOTE: Having a number after the space appears to be important. I tried a letter after the space ( Example: "Mk b" ), and it did not cause problems for sorting.
